config/breadcrumbs.rb

```diff
@@ -99,6 +99,11 @@
   parent :editorial_root
 end
 
+crumb :new_asset do
+  link 'Create asset', new_editorial_asset_path
+  parent :editorial_root
```

Comment:
It may or may not make more sense to use the asset index page as the parent for the new asset page.